packages/writing-style/styles/commercetools/DiscriminatoryWords.yml

```diff
@@ -0,0 +1,11 @@
+# This list contains expressions that are discriminating which should not be used.
+extends: existence
+message: "'%s' is a discriminating expression. Please use a different wording."
+ignorecase: true
+level: error
+tokens:
+  - whitelist(?:ed|)
+  - blacklist(?:ed|)
+  - master
+  - slave
+  - native
```

Comment:
please move the existing "disable" here too and consider making this rather  a replacement rule like WordList because you can then suggest the recommeded alternative term.

Reply:
I considered it and my feeling was that in some cases it is difficult to suggest the right alternative since it depends strongly on the context and what the writer wants to express with the word. For the first two it is easy, yes. But I think for the other words it is more difficult.